aspire-update.php

```diff
@@ -58,3 +58,22 @@ function aspire_update_uninstall_hook() {
 	$admin_settings = AspireUpdate\Admin_Settings::get_instance();
 	$admin_settings->delete_all_settings();
 }
+
+// Load and start translations updater.
+require_once __DIR__ . '/vendor/afragen/autoloader/Autoloader.php';
+new Fragen\Autoloader( [ 'Fragen\\Translations_Updater' => __DIR__ . '/vendor/afragen/translations-updater/src/Translations_Updater' ] );
+add_action(
+	'admin_init',
+	function () {
```

Comment:
Like the hooks above, can we abstract this to a function like `aspireupdate_init_translations()`?

Reply:
Sure.

Reply:
done.